client/state/user-settings/reducer.js

```diff
@@ -57,8 +57,22 @@ export const updatingPassword = ( state = false, action ) => {
 	return state;
 };
 
+export const updating = ( state = false, action ) => {
+	switch ( action.type ) {
+		case USER_SETTINGS_SAVE: {
```

Comment:
I realize those were here before, but just wanted to point out that it can be confusing what `SAVE` and what `UPDATE` means in this context. Asking those questions adds to the confusion: why is it `UPDATE_FAILURE` and not `SAVE_FAILURE`?

Reply:
I agree, those are relics from the past. Will make sure to properly rename them and avoid confusion.

Reply:
fixed in ebfd7f1